packages/daimo-api/src/db/db.ts

```diff
@@ -179,6 +179,149 @@ export class DB {
 
           UNIQUE (address, push_key)
         );
+
+        --
+        -- DaimoDB Index setup (filtered from Jumbo)
+        --
+        CREATE TABLE IF NOT EXISTS index.daimo_index (
+          chain_id BIGINT NOT NULL,
+          latest_block_num BIGINT NOT NULL, 
+        );
+
+        CREATE TABLE IF NOT EXISTS index.daimo_transfers (
+          chain_id NUMERIC NOT NULL,
+          block_num NUMERIC NOT NULL,
+          block_hash BYTEA NOT NULL,
+          block_ts NUMERIC NOT NULL,
+          tx_idx NUMERIC NOT NULL,
+          tx_hash BYTEA NOT NULL,
+
+          sort_idx NUMERIC NOT NULL,
+          token BYTEA NOT NULL, -- 0x0 for native token transfers
+          f BYTEA NOT NULL,
+          t BYTEA NOT NULL,
+          amount NUMERIC NOT NULL
+        );
+
+        CREATE TABLE IF NOT EXISTS index.daimo_tx (
+          chain_id NUMERIC NOT NULL,
+          block_num NUMERIC NOT NULL,
+          block_hash BYTEA NOT NULL,
+          block_ts NUMERIC NOT NULL,
+          tx_idx INT NOT NULL,
+          tx_hash BYTEA NOT NULL,
+        )
+
+        CREATE TABLE IF NOT EXISTS index.daimo_op (
+          chain_id NUMERIC NOT NULL,
+          block_num NUMERIC NOT NULL,
+          block_hash BYTEA NOT NULL,
+          block_ts NUMERIC NOT NULL,
+          tx_idx NUMERIC NOT NULL,
+          tx_hash BYTEA NOT NULL,
+          log_idx NUMERIC NOT NULL,
+          log_addr NUMERIC NOT NULL,
+          event_name BYTEA NOT NULL,
+
+          op_hash BYTEA NOT NULL,
+          op_sender BYTEA NOT NULL,
+          op_paymaster BYTEA NOT NULL,
+          op_nonce NUMERIC NOT NULL,
+          op_success BOOLEAN NOT NULL,
+          op_actual_gas_cost NUMERIC NOT NULL,
+          op_actual_gas_used NUMERIC NOT NULL
+        );
+
+        CREATE TABLE IF NOT EXISTS index.daimo_request (
+          chain_id NUMERIC NOT NULL,
+          block_num NUMERIC NOT NULL,
+          block_hash BYTEA NOT NULL,
+          block_ts NUMERIC NOT NULL,
+          tx_idx NUMERIC NOT NULL,
+          tx_hash BYTEA NOT NULL,
+          log_idx NUMERIC NOT NULL, 
+          log_addr BYTEA NOT NULL,
+          event_name BYTEA NOT NULL
+
+          id NUMERIC NOT NULL,
+          recipient BYTEA,
+          creator BYTEA,
+          amount NUMERIC,
+          metadata BYTEA,
+          canceller BYTEA,
+          fulfiller BYTEA,
+        ); 
+
+        CREATE TABLE IF NOT EXISTS index.daimo_note (
+          chain_id BIGINT NOT NULL,
+          block_num BIGINT NOT NULL,
+          block_hash BYTEA NOT NULL,
+          block_ts NUMERIC NOT NULL,
+          tx_idx NUMERIC NOT NULL,
+          tx_hash BYTEA NOT NULL,
+          log_addr BYTEA NOT NULL,
+          event_name BYTEA NOT NULL,
```

Comment:
this one's missing log_idx

i'd put this same block at the top of every events table:

```sql
          chain_id NUMERIC NOT NULL,
          block_num NUMERIC NOT NULL,
          block_hash BYTEA NOT NULL,
          block_ts NUMERIC NOT NULL,
          tx_idx NUMERIC NOT NULL,
          tx_hash BYTEA NOT NULL,
          log_idx NUMERIC NOT NULL, 
          log_addr BYTEA NOT NULL,
          event_name BYTEA NOT NULL
```